std_execution.bs

```diff
@@ -4577,21 +4577,21 @@ template &lt;class E>  // arguments are not associated entities ([lib.tmpl-heads
 
         where <code><i>connect-awaitable-promise</i></code> is the promise type of <code><i>connect-awaitable</i></code>, and where <code><i>connect-awaitable</i></code> suspends at the <i>initial suspends point</i> ([dcl.fct.def.coroutine]), and:
 
-          1. <i>set-value-expr</i> first evaluates `co_await (S&&) s`, then suspends the coroutine and evaluates `execution::set_value((R&&) r)` if <code><i>await-result-type</i>&lt;S, <i>connect-awaitable-promise</i>></code> is <code><i>cv</i> void</code>; otherwise, it evaluates `auto&& res = co_await (S&&) s`, then suspends the coroutine and evaluates `execution::set_value((R&&) r, (decltype(res)) res)`.
+          1. <i>set-value-expr</i> first evaluates `co_await std::forward<S>(s)`, then suspends the coroutine and evaluates `execution::set_value(std::forward<R>(r))` if <code><i>await-result-type</i>&lt;S, <i>connect-awaitable-promise</i>></code> is <code><i>cv</i> void</code>; otherwise, it evaluates `auto&& res = co_await std::forward<S>(s)`, then suspends the coroutine and evaluates `execution::set_value(std::forward<R>(r), std::forward<decltype(res)>(res))`.
```

Comment:
Can't tell from my phone, but if memory serves these should be moves, not forwards. I could be totally wrong though.

Reply:
Ah yes, because this is within a function that accepts them with decayed types. Will fix.